cirq/study/result_test.py

```diff
@@ -300,6 +301,25 @@ def test_json_bit_packing_and_dtype():
     np.testing.assert_allclose(len(bits_json), len(digits_json) / 8, rtol=0.02)
 
 
+def test_json_bit_packing_error():
+    with pytest.raises(ValueError):
+        _pack_digits(np.ones(10), pack_bits='hi mom')
+
+
+def test_json_bit_packing_force():
+    assert _pack_digits(np.ones(10, dtype=int), pack_bits='force') == _pack_digits(
+        np.ones(10), pack_bits='auto'
+    )
+
+    assert _pack_digits(2 * np.ones(10, dtype=int), pack_bits='force') != _pack_digits(
+        2 * np.ones(10, dtype=int), pack_bits='auto'
+    )
+    # These are the `np.packbits` semantics; not sure I agree with them:
```

Comment:
Your opinion is duly noted :) however I think a little explanation here could be useful for the readers!

Reply:
added an explanation